docs/ProductionChecklistForScalarDLAuditor.md

```diff
@@ -0,0 +1,172 @@
+
+{% capture notice--info %}
+**Note**
+
+If you place the worker nodes in different availability zones (AZs), you can withstand an AZ failure.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### Worker node specifications
+
+From the perspective of commercial licenses, resources for one pod running ScalarDL Auditor are limited to 2vCPU / 4GB memory. In addition, some pods other than ScalarDL Auditor pods exist on the worker nodes.
+
+In other words, the following components could run on one worker node:
+
+* ScalarDL Auditor pod (2vCPU / 4GB)
+* Envoy proxy
+* Monitoring components (if you deploy monitoring components such `kube-prometheus-stack`)
+* Kubernetes components
+
+With this in mind, you should use a worker node that has at least 4vCPU / 8GB memory resources and use at least three worker nodes for availability, as mentioned in [Number of pods and Kubernetes worker nodes](#number-of-pods-and-kubernetes-worker-nodes).
+
+However, three nodes with at least 4vCPU / 8GB memory resources per node is the minimum for a production environment. You should also consider the resources of the Kubernetes cluster (for example, the number of worker nodes, vCPUs per node, memories per node, and ScalarDL Auditor pods), which depend on your system's workload. In addition, if you plan to scale the pods automatically by using some features like [Horizontal Pod Autoscaling (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/), you should consider the maximum number of pods on the worker node to decide on the worker node resources.
+
+### Network
+
+You should create the Kubernetes cluster on a private network since ScalarDL Auditor does not provide any services to users directly via internet access. We recommend accessing ScalarDL Auditor via a private network from your applications.
+
+### Monitoring and logging
+
+You should monitor the deployed components and collect their logs. For details, see [Monitoring Scalar products on a Kubernetes cluster](./K8sMonitorGuide.md) and [Collecting logs from Scalar products on a Kubernetes cluster](./K8sLogCollectionGuide.md).
+
+### Backup and restore
+
+You should enable the automatic backup feature and point-in-time recovery (PITR) feature in the backend database. For details, see [Set up a database for ScalarDB/ScalarDL deployment](./SetupDatabase.md).
+
+### Do not deploy ScalarDL Auditor on the same Kubernetes cluster as ScalarDL Ledger
+
+For Byzantine fault detection in ScalarDL to work properly, do not deploy ScalarDL Auditor pods on the same Kubernetes clusters as the ScalarDL Ledger deployment. Please deploy ScalarDL Auditor pods on an environment other than the administrative domain (other than the Kubernetes cluster) for ScalarDL Ledger deployment.
+
+
+#### Required for production environments
+
+```mermaid
+graph LR
+  subgraph "ScalarDL Auditor mode"
```

Comment:
```suggestion
  subgraph "ScalarDL"
```
This subgraph includes Ledger and Auditor, so naming it `Auditor mode` is probably misleading.

Reply:
Ah, I see. You are right. It's a bit confusing.
I will apply your suggestion! Thank you!